src/pokemon_debug.c

```diff
@@ -0,0 +1,740 @@
+//Credits: Gamer2020, AsparagusEduardo, TheXaman, ShinyDragonHunter
+#include "global.h"
+#include "battle.h"
+#include "battle_anim.h"
+#include "bg.h"
+#include "constants/rgb.h"
+#include "constants/songs.h"
+#include "data.h"
+#include "decompress.h"
+#include "field_weather.h"
+#include "gpu_regs.h"
+#include "item.h"
+#include "item_icon.h"
+#include "list_menu.h"
+#include "m4a.h"
+#include "main.h"
+#include "malloc.h"
+#include "menu.h"
+#include "overworld.h"
+#include "palette.h"
+#include "palette_util.h"
+#include "pokemon.h"
+#include "pokemon_animation.h"
+#include "pokemon_debug.h"
+#include "pokemon_icon.h"
+#include "reset_rtc_screen.h"
+#include "scanline_effect.h"
+#include "script.h"
+#include "script_pokemon_util.h"
+#include "sound.h"
+#include "string_util.h"
+#include "strings.h"
+#include "task.h"
+#include "trainer_pokemon_sprites.h"
+
+#include "constants/items.h"
+
+//Defines
+#define DEBUG_MON_X 137
+#define DEBUG_MON_Y 18
+#define DEBUG_MON_BACK_X 32
```

Comment:
I messed up and accidentally moved the coords in the wrong direction when submitting the changes, but this should fix it. I also fully tested what X value for the backsprite is needed for syncing the back sprite to how it is in battle.
```suggestion
#define DEBUG_MON_X 144
#define DEBUG_MON_Y 11
#define DEBUG_MON_BACK_X 40
```